sympy/utilities/lambdify.py

```diff
@@ -8,7 +8,12 @@
 from functools import wraps
 import inspect
 import textwrap
-
+from sympy import *
+# from sympy import Derivative
+from sympy import Integer
+from sympy import diff
+from sympy import sin
+from sympy import cos
```

Comment:
Usually (look around in SymPy codebase) the import sources are more specific than that:
```
from sympy.core.numbers import Integer
from sympy.core.function import diff
from sympy.functions.elementary.trigonometric import sin, cos
```